actions/promote_package.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: promote_package
+runner_type: mistral-v2
+description: Promote a package to production from staging.
+enabled: true
+entry_point: workflows/promote_package.yaml
+parameters:
+  package:
+    type: string
+    description: Package to promote
+    required: true
+  version:
+    type: string
+    description: Version to promote
+    required: true
+  revision:
+    type: string
+    description: Revision to promote
+    required: true
+  os:
+    description: OS to promote (will promote all if not specified)
```

Comment:
Need examples. Technically these are enums, so either specify enums or give samples of what I should type here. E.g. do I type ubuntu, Ubuntu, Ubuntu12.04, deb, Deb?

Same applies to version, and revision. Is it v1.3, 1.3, 1.3.6, etc? 

For revision: in app other workflows `revision` == git sha. In case of packages, it's likely the last number of build, which is 1.3.2-17. So the choice of word is confusing, please find a right one.